synapse/storage/databases/main/events.py

```diff
@@ -52,16 +52,6 @@
 )
 
 
-def encode_json(json_object):
-    """
-    Encode a Python object as JSON and return it in a Unicode string.
-    """
-    out = frozendict_json_encoder.encode(json_object)
-    if isinstance(out, bytes):
-        out = out.decode("utf8")
```

Comment:
This was dealing with py2 encoder logic, which made this function fairly useless so I figured it made more sense to inline it.